app/components/CallCharacter/index.tsx

```diff
@@ -264,13 +264,15 @@ export function CallCharacter({ character }: { character: CharacterType }) {
 
     console.log(`CallCharacter: created voice session`);
     session.warmup();
-    session.startAudio(); // This will prompt for mic permission.
-
-    setVoiceSession(session);
-    // Wait a beat before starting the ringtone.
-    setTimeout(() => {
-      ringtone.play();
-    }, 1000);
+    // This will prompt for mic permission.
+    session.startAudio().then(() => {
+      // Starting audio session.
+      setVoiceSession(session);
+      // Wait a beat before starting the ringtone.
+      setTimeout(() => {
```

Comment:
why not start right away now? or does it feel too jarring?

Reply:
It's too jarring because real phones don't start it immediately.